src/like-service/app/Http/Controllers/LikeController.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace app\Http\Controllers;
+
+use App\Models\User;
+use Illuminate\Foundation\Auth\Access\AuthorizesRequests;
+use Illuminate\Foundation\Bus\DispatchesJobs;
+use Illuminate\Foundation\Validation\ValidatesRequests;
+use Illuminate\Routing\Controller as BaseController;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Http;
+use Illuminate\Support\Facades\Log;
+
+class LikeController extends BaseController
+{
+    use AuthorizesRequests, DispatchesJobs, ValidatesRequests;
+
+    function doLike($request)
+    {
+        $user_token = $request->cookie('jwt');
+        $postId = $request->header('postId');
+
+        if (!$this->validateToken($user_token)) {
+            return response()->json([
+                'message' => 'Unauthorized'
+            ], 401);
+        } else {
+            $userId = $this->extractUserIdFromToken($user_token);
```

Comment:
[NIT] This does not need to be inside the `else` block. We can move it out, because it is returned for the unautherized case

Reply:
else-block removed from all 3 functions

Reply:
[NIT] Please reformat again :)